recipes/savana/meta.yaml

```diff
@@ -25,7 +27,7 @@ requirements:
     - pybedtools >=0.9.0
     - pysam >=0.20.0
     - cyvcf2 >=0.30.16
-    - scikit-learn >=1.2.2
+    - scikit-learn >=1.2.2,<1.3.0
```

Comment:
Hi @dpryan79 I've just tested installing the newly merged `1.0.4` version of savana, with both `conda` and `mamba`, but version `1.3.0` of scikit-learn is still being installed though I've tried to pin it below `1.3.0` in the `meta.yaml` here. Would you be able to advise how best to pin the version of a dependency? Many thanks.